locations/spiders/travelex.py

```diff
@@ -9,7 +9,7 @@ class TravelexSpider(scrapy.Spider):
     allowed_domains = ["https://www.travelex.co.uk/"]
 
     def start_requests(self):
-        countries = ['dech', 'au', 'gb', 'enbh', 'de', "nl", "zhhk", "jajp", "my", "nz", "qa"]
+        countries = ['dech', 'au', 'gb', 'enbh', 'de', "zhhk", "jajp", "my", "nz", "qa"]
```

Comment:
`om`, `ae` can be added. There are a few more countries for which I couldn't figure out the right language/country code.

`nl` should probably be added back here again as it uses the same API? I'm not sure what the purpose of splitting gwk_travelex into a separate spider? The `brand` and `brand_wikidata` attributes can then be modified for `nl` as required to use the alternate brand `GWK Travelex`.

Reply:
i'll try that !